auslib/test/admin/views/test_releases.py

```diff
@@ -323,6 +336,18 @@ def testReleasePostUpdateOutdatedDataNotBlob(self):
                                                    name='ee', product='ee', data_version=1))
         self.assertStatusCode(ret, 400)
 
+        blob = json.loads(blob)
+        history_rows = dbo.releases.history.t.select().where(dbo.releases.history.name == "ee").execute().fetchall()
+        self.assertEqual(len(history_rows), 4)
+        self.assertEqual(history_rows[0]["data"], None)
+        self.assertEqual(history_rows[0]["read_only"], False)
+        self.assertEqual(history_rows[1]["data"], {"name": "ee", "schema_version": 1, "hashFunction": "sha512"})
+        self.assertEqual(history_rows[1]["read_only"], False)
```

Comment:
Items 0 and 1 are created as part of the initial _post. Whenever a new object is added we end up with a row in the history table that is all null except for the PK and history metadata (https://github.com/mozilla/balrog/blob/bc70d0f8c9675f99407b276bb4b449f7fa84f471/auslib/db.py#L693).

Reply:
The fact that the initial _post passes data_version is probably part of the confusion here. It shouldn't be doing that since it's postihng a new release...

Reply:
And I'd completely forgotten (until further debugging) that creating a release with this type of _post ends up creating it, then updating it -- so the first 3 items of the history_rows are all created by the first _post, and then the final one by the second.